app/controllers/admin/domain_allows_controller.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
+require 'csv'
+
 class Admin::DomainAllowsController < Admin::BaseController
   before_action :set_domain_allow, only: [:destroy]
+  before_action :dummy_import, only: [:new, :create]
```

Comment:
Is that required?

Reply:
Yes, otherwise rails raises errors when processing the haml